test/specs/global.deprecations.tests.js

```diff
@@ -6,6 +6,32 @@ describe('Deprecations', function() {
 				expect(Chart.layoutService).toBe(Chart.layouts);
 			});
 		});
+
+		describe('Legend Labels: usePointStyle option', function() {
```

Comment:
@simonbrunel should we put this inside a `describe('Version 2.8.0', ...)` block?

Reply:
It's already inside 2.8.0 deprecations